### PR TITLE
Update katsu to the v3.0 data model part 2

### PR DESCRIPTION
**Pull request by SonQBChau**
## What's new
- Continue from part 1
- Move factories to a new location
- Update factory model to v3
- Remove the old test model
- Rewrite the test model

### How to test
- Just code review

### CODE REVIEW

**Review by mshadbolt — APPROVED** [review]
Did quick overview and looks good. Mostly looked at factories

**Review by kcranston — APPROVED** [review]

**Comment by kcranston** [comment]
I assume the failing tests are because of dependencies on pr #246

File: chord_metadata_service/mohpackets/tests/models/test_biomarker.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.test import TestCase
+
+from chord_metadata_service.mohpackets.models import Biomarker, Program
+from chord_metadata_service.mohpackets.permissible_values import HER2_STATUS
+from chord_metadata_service.mohpackets.schemas.model import DonorModelSchema
+from chord_metadata_service.mohpackets.tests.factories import BiomarkerFactory
+from chord_metadata_service.mohpackets.tests.utils import INVALID_ID_LIST
```
Comment (OrdiNeu) [inline chord_metadata_service/mohpackets/tests/models/test_biomarker.py:8]: This and DonorModelSchema is unused -- either the test should use it or the import should be removed

**Review by OrdiNeu — APPROVED** [review]

File: chord_metadata_service/mohpackets/tests/models/test_biomarker.py (hunk 1 above)
Comment (SonQBChau) [reply chord_metadata_service/mohpackets/tests/models/test_biomarker.py:8]: yea, I will fix all the linter errors on final

**Pull request merged**